awx/main/models/events.py

```diff
@@ -192,6 +192,8 @@ class Meta:
     EVENT_CHOICES = [(x[1], x[2]) for x in EVENT_TYPES]
     LEVEL_FOR_EVENT = dict([(x[1], x[0]) for x in EVENT_TYPES])
 
+    skip_save = False
```

Comment:
What will the final set of counters look like for jobs that have events that are ignored with this toggle? Will they still be continuous?